mava/utils/make_env.py

```diff
@@ -42,6 +44,11 @@
     "LevelBasedForaging-v0": {"generator": LbfRandomGenerator, "wrapper": LbfWrapper},
 }
 
+# Define a different registry for Matrax since it has no generator.
+_matrax_registry = {
+    "Matrax": {"wrapper": MatraxWrapper},
+}
```

Comment:
Because we only have 1 wrapper we don't need a registry for matrax :smile: 
```suggestion
```

Reply:
I think the `make_matrax_env` and the `make_jumanji_env` methods are basically the same, so what we could do is join the matrax and jumanji registries and then just use the `make_jumanji_env` methods. 

@RuanJohn @JemmaLDaniel what do you think?

Reply:
That is definitely more concise, but I think keeping them separate is slightly easier to read and understand because it is in keeping with the choice for Matrax as a separate suite of environments to Jumanji. Another solution is to merge them into one registry but change the name to be more all-encompassing than `make_jumanji_env` and `_jumanji_registry`?

Reply:
I agree that the methods are essentially the same, but I think it is better to keep them separate and to have a matrax registry. The reason for this is that the poly games may become part of matrax later on keeping env suites separate from each other sets a nice pattern.

Reply:
Ok sure happy to keep with this pattern :smile: